spec/models/user_spec.rb

```diff
@@ -963,15 +963,15 @@ def user_from_authorization_service(service_name, signed_in_resource, cta_varian
     it "returns the number of articles read per day" do
       user.created_at = Time.zone.today.days_ago(4)
       create_list(:page_view, 10, user: user)
-      expect(user.number_articles_per_day).to eq(2.5)
+      expect(user.number_articles_per_day).to eq(3)
```

Comment:
removed the decimal place for readability and because it does not contribute useful information